plexapi/video.py

```diff
@@ -925,13 +962,48 @@ def _loadData(self, data):
         self.thumbAspectRatio = data.attrib.get('thumbAspectRatio')
         self.viewOffset = utils.cast(int, data.attrib.get('viewOffset', 0))
         self.year = utils.cast(int, data.attrib.get('year'))
-
+        
     @property
     def locations(self):
         """ This does not exist in plex xml response but is added to have a common
             interface to get the locations of the clip.
 
-            Retruns:
+            Returns:
                 List<str> of file paths where the clip is found on disk.
         """
         return [part.file for part in self.iterParts() if part]
+
+    def section(self):
+        """Return the :class:`~plexapi.library.LibrarySection` this item belongs to."""
+        # Clip payloads currently do not contain 'librarySectionID'.
+        # Return None to avoid unnecessary attribute lookup attempts.
+        return None
+
+
+@utils.registerPlexObject
```

Comment:
Don't register this object.

Reply:
Actually, we should just combine all the `Extra` attributes into `Clip`. `Extra` is not needed.